### DISCUSSION TITLE
Delete blocks with keys

### DISCUSSION THREAD

**Discussion by IonCuceritorul** [ScratchAddons/ScratchAddons#5451] [Ideas]
It would be nice to have a combination of keys (ex: "shift" + "enter"), or just press the "del" key to delete clocks.

**Comment by BroJac5246** [comment]
How would we know what block(s) to delete?

**Comment by CST1229** [comment]
You can already delete the last clicked/dragged block with Delete or Backspace.

**Comment by DNin01** [comment]
You can already delete the block that you last clicked on by pressing Delete, but there could be an addon that deletes the block your mouse is hovering over instead.
